srcpkgs/qt5-go/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'qt5-go'
+pkgname=qt5-go
+version=0.1
+revision=1
+archs="*"
+wrksrc=qt-b56356132618ecc7c0029f4262d4ac986afe1477
```

Comment:
the hash should be a variable, like `_githash` and then when needed use `${_githash}` instead to avoid repetition